+Prefs/+Inputs/LabelControlBasic.m

```diff
@@ -35,6 +41,11 @@
             % Here, widths will all be taken care of in adjust_UI
             tag = strrep(pref.name,' ','_');
             labeltext = obj.get_label(pref);
+            
+            if strcmp(obj.uistyle, 'checkbox') && strcmp(pref.units, '0/1')
+                labeltext = labeltext(1:end-6);
+            end
+            
```

Comment:
Not sure if there's a better way to do this, but I don't want the default units to display beside checkboxes.

Reply:
why not?

Reply:
"Label [0/1] [x]" <--checkbox is redundant because it's obviously 0/1 if there's a checkbox. This bothers me.